core/controllers/question_test.py

```diff
@@ -243,29 +118,29 @@ def test_put(self):
         response = self.testapp.get('/preferences')
         csrf_token = self.get_csrf_token_from_response(response)
         response_json = self.put_json(
-            '%s/%s/%s' % (
-                feconf.QUESTION_DATA_URL, self.collection_id, question_id),
+            '%s/%s' % (
+                feconf.QUESTION_DATA_URL, question_id),
             payload, csrf_token, expect_errors=False)
         self.assertIn('question_id', response_json.keys())
 
         del payload['change_list']
         self.put_json(
-            '%s/%s/%s' % (
-                feconf.QUESTION_DATA_URL, self.collection_id,
+            '%s/%s' % (
+                feconf.QUESTION_DATA_URL,
                 question_id), payload, csrf_token, expect_errors=True,
             expected_status_int=404)
 
         del payload['commit_message']
         payload['change_list'] = json.dumps(change_list)
         self.put_json(
-            '%s/%s/%s' % (
-                feconf.QUESTION_DATA_URL, self.collection_id,
+            '%s/%s' % (
+                feconf.QUESTION_DATA_URL,
                 question_id), payload, csrf_token, expect_errors=True,
             expected_status_int=404)
 
         payload['commit_message'] = 'update title'
         self.put_json(
-            '%s/%s' % (feconf.QUESTION_DATA_URL, self.collection_id),
+            '%s' % (feconf.QUESTION_DATA_URL),
```

Comment:
You don't even need the interpolation here, just use feconf.QUESTION_DATA_URL directly

Reply:
Done, removed all occurrences of this.